runtime/centrifuge/src/lib.rs

```diff
@@ -1275,11 +1275,9 @@ pub struct PoolCurrency;
 impl Contains<CurrencyId> for PoolCurrency {
 	fn contains(id: &CurrencyId) -> bool {
 		match id {
-			CurrencyId::Tranche(_, _)
-			| CurrencyId::Native
-			| CurrencyId::KSM
-			| CurrencyId::Staking(_) => false,
-			CurrencyId::AUSD => true,
+			// Deprecated, remove in the next runtime upgrade
```

Comment:
See comment above about potentially specifying the spec version here as we can expect the migration to be part of `1020`.